https/gifts.js

```diff
@@ -5,113 +5,56 @@ also has functions to return admin stuff as well
 */
 const functions = require('firebase-functions');
 require('firebase/firestore');
-const { getUUID, getGeneralQueries } = require('../utils/utils');
-const { YEAR } = require("../config/constants");
+const { getUUID, getGeneralQueries, markGifteeAccount } = require('../utils/utils');
+const { EVENT } = require("../config/constants");
 const db = require('../config/db');
 
 // marks gift as went
-const sendGift = functions.https.onCall(async({user}, context) => {
-  // marks their own account
+const sendGift = functions.https.onCall(async({user, value, giftee_uuid}, context) => {
+  // this has to mark both the giftee and gifter
 
+  if(typeof value === "undefined"){value = true}
```

Comment:
You can set a default value in line 13 like this: `...async({user, value = true, giftee_uuid}...` .

Reply:
TIL, actually a really nice way to deal with it